docs/project/goals.md

```diff
@@ -76,17 +78,17 @@ community to be welcoming and respectful, with a deep commitment to
 psychological safety. We need consistent expectations for how every community
 member should behave, regardless of their position in the community. These
 expectations around conduct and behavior need to be clearly articulated both to
-set expectations for people joining, and to help remind and anchor us on
+set expectations for people joining and to help remind and anchor us on
```

Comment:
```suggestion
set expectations for people joining, and to help remind and anchor us on
```

I view the comma here as stylistic: there's two `and`s in this sentence so having the comma here can help clarify the intended reading. There might be some better rephrasing, but it's probably better to keep the comma for now.